src/App.tsx

```diff
@@ -1,16 +1,74 @@
 import React from 'react';
 import './App.scss';
 
-class App extends React.Component {
-  addOne = () => {};
+type State = {
+  count: number,
+};
 
-  add100 = () => {};
+class App extends React.Component<{}, State> {
+  state = {
+    count: 0,
+  };
 
-  increase = () => {};
+  addOne = () => {
+    this.setState((state) => {
+      return {
+        count: state.count + 1,
+      };
+    });
+  };
+
+  add100 = () => {
+    this.setState((state) => {
+      return {
+        count: state.count + 100,
+      };
+    });
+  };
+
+  increase = () => {
+    if (this.state.count % 5 === 0) {
+      this.add100();
+      this.addOne();
+    } else {
+      this.addOne();
+    }
```

Comment:
don't repeat yourself
```suggestion
    if (this.state.count % 5 === 0) {
      this.add100();
    }
    
    this.addOne();
```